### DISCUSSION TITLE
Dynamic values in Git Message settings?

### DISCUSSION THREAD

**Discussion by balaji-dutt** [estruyf/vscode-front-matter#446] [Q&A] [answered]
Hi,

Just started using this extension and am glad to be able to move yet another thing to VSCode 😄 

Does the Git Message setting support the use of variables that get replaced by actual values when I commit? For example,  I would like to have the commit message read `Committing posts made on <DD-MM-YYYY HH:MM>` or even something like `Committing posts as of {{now}}` and have the timestamps populated automatically.

**Comment by lorezyra** [comment]
Do you mean that you want FM to do the `git commit` for you?

**Reply by balaji-dutt** [reply]
I guess I'm confused now. My assumption was the "sync" option that appears when you enable`frontMatter.git.enabled` (as shown [here](https://frontmatter.codes/docs/git-integration#enable-git-integration)) did generate a commit. Otherwise I'm not sure what the `frontMatter.git.commitMessage` is used for.

**Reply by estruyf** [reply]
The sync indeed creates a commit and pushes it.

Currently the message is a static, we can start thinking to support placeholders like `{{now}}`.

**Reply by balaji-dutt** [reply]
Cool! Should I raise a [Enhancement] request via the Issues tab?

**Reply by lorezyra** [reply]
Now that I think of this, it would make my workflow a lot easier… As soon as I do the final publishing of a post, it should do a commit. That would be ideal!

**Reply by balaji-dutt** [reply]
New enhancement request created! [Enhancement: Support for dynamic values in Front Matter › Git: Commit Message?](https://github.com/estruyf/vscode-front-matter/issues/447)

**Comment by estruyf** [comment, marked as answer]
Like @balaji-dutt mentioned, this has been implemented in #447 

More information how to use it can be read here: https://frontmatter.codes/docs/git-integration#change-the-commit-message
